PluralKit.Bot/Commands/Member.cs

```diff
@@ -73,7 +73,23 @@ public async Task ViewMember(Context ctx, PKMember target)
             var system = await _db.Execute(c => _repo.GetSystem(c, target.System));
             await ctx.Reply(embed: await _embeds.CreateMemberEmbed(system, target, ctx.Guild, ctx.LookupContextFor(system)));
         }
+        public async Task AddReminder(Context ctx, PKMember target) {
+            if (ctx.System?.Id == target.System) {
+                await using var conn = await _db.Obtain();
+                await _repo.AddReminder(conn, new PKReminder { Mid = ctx.Message.Id, Channel = ctx.Channel.Id, Guild = ctx.Guild == null ? null : ctx.Guild.Id, Member = target.Id, System = target.System, Seen = false });
+                await ctx.Reply($"Added new reminder for {target.Name}");
+            }
+            else {
+                await ctx.Reply($"{Emojis.Error} You can only send reminders to your own system.");
+            }
+        }
 
+        public async Task GetReminders(Context ctx, PKMember target) => await ctx.RenderReminderList(
+            _db,
+            target.Id,
+            $"Reminders for {target.Name}",
+            target.Color,
+            true);
```

Comment:
I'm pretty much nitpicking at this point, but could you add a newline here?
```suggestion
            true);

```